packages/cli/src/commands/config/index.ts

```diff
@@ -669,15 +670,16 @@ export const commandsConfig: CommandsConfig = {
     ],
   },
   decode: {
-    description: 'decode transaction data using the ABIs of the given Cannon package',
+    description:
+      'Decode the given data using the ABIs from the specified Cannon package. It will try to parse the data as Function Call, Error or Event. If a tx hash is given it will fetch it using the rpc and try to parse the input.',
```

Comment:
I'm pretty sure we don't support txn hashes on this... unless this feature was added recently and I didnt realize?